jujugui/static/gui/src/app/components/deployment-flow/budget/budget.js

```diff
@@ -127,6 +141,7 @@ YUI.add('deployment-budget', function() {
               <juju.components.InsetSelect
                 disabled={disabled}
                 label="Budget"
+                onChange={this._handleBudgetChange}
```

Comment:
I'd kind of figure we'd only want to set the budget change on a 'save' but I'll leave that up to to the design